aziotctl/src/internal/check/checks/well_formed_configs.rs

```diff
@@ -171,14 +189,71 @@ impl Checker for WellFormedIdentitydConfig {
 enum DaemonCfg<T> {
     Cfg(T),
     PermissionDenied(Error),
+    IncorrectPermissions(Error),
+    IncorrectGroupId(Error),
+    IncorrectUserID(Error),
 }
 
 async fn load_daemon_cfg<T: serde::de::DeserializeOwned>(
     daemon: &str,
+    uid: &str,
     config_path: &Path,
     config_directory_path: Option<&Path>,
     shared: &CheckerShared,
 ) -> Result<DaemonCfg<T>> {
+    use std::os::linux::fs::MetadataExt;
+    if let Ok(metadata) = fs::metadata(config_path) {
```

Comment:
This is wrong for two reasons:

1. `config_path` doesn't necessarily exist. In fact the situation that it doesn't exist is the default - `config apply` does not create it.

1. None of the files in `config_directory_path` are being checked.

Reply:
Then what can we check? I the PR useful? I set up a meeting to discuss with you.